flutter_app/gallery_app_new/lib/main.dart

```diff
@@ -1,21 +1,40 @@
 import 'package:flutter/material.dart';
 import 'package:gallery_app_new/modules/home/screen/home_screen.dart';
+import 'package:gallery_app_new/modules/text/screen/text_screen.dart';
+import 'package:go_router/go_router.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(MyApp());
 }
 
 class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+  MyApp({super.key});
+
+  final _router = GoRouter(
+    initialLocation: '/',
+    routes: [
+      GoRoute(
+        name: 'home',
+        path: '/',
+        builder: (context, state) => const HomeScreen(),
+      ),
+      GoRoute(
+        name: 'richText',
+        path: '/richText',
+        builder: (context, state) => const TextScreen(),
+      ),
+    ],
+  );
```

Comment:
Please refer to [example](https://github.com/GoldenOwlAsia/mobile-flutter-template/blob/development/lib/src/router/router.dart) to refactor the code